src/components/Slider/SliderFixedRange.tsx

```diff
@@ -0,0 +1,334 @@
+      const rightProgressRatio = rightProgressing / range
+      leftAnimated.value = {zIndex: INVISIBLE, opacity: INVISIBLE}
+      rightAnimated.value = {zIndex: VISIBLE, opacity: VISIBLE}
+
+      // When sliding the thumb across a distance shorter than the track's width
+      if (rightProgressing < leftProgress.value) {
+        runOnJS(updateSlider)(leftProgress.value, left, ThumbPosition.right)
+      }
+      // When sliding the thumb over the track's width
+      else if (rightProgressing > sliderWidth) {
+        runOnJS(updateSlider)(sliderWidth, totalPoint, ThumbPosition.right)
+      }
+      // When sliding steadily increases
+      else if (rightProgressing > range * (right + NEXT_STEP)) {
+        const currentProgress = range * Math.floor(rightProgressRatio)
+        const point = Math.floor(rightProgressRatio)
+        runOnJS(updateSlider)(currentProgress, point, ThumbPosition.right)
+      }
+      // When sliding steadily decreases
+      else if (rightProgressing < range * (right - PREVIOUS_STEP)) {
+        const currentProgress = range * Math.ceil(rightProgressRatio)
+        const point = Math.ceil(rightProgressRatio)
+        runOnJS(updateSlider)(currentProgress, point, ThumbPosition.right)
+      }
+    },
+    onEnd: () => {
+      // This line sets the opacity of the right label animated
+      rightAnimated.value = {...rightAnimated.value, opacity: INVISIBLE}
+      runOnJS(onValueChange)({
+        minimum: minimumValue + sliderValue.value.left * step,
+        maximum: minimumValue + sliderValue.value.right * step,
+      })
+    },
+  })
+
+  const leftThumbRangeStyle = useAnimatedStyle(() => ({
+    transform: [{translateX: withTiming(leftProgress.value, {duration: 1})}],
+    zIndex: leftAnimated.value.zIndex,
+  }))
+
+  const rightThumbRangeStyle = useAnimatedStyle(() => ({
+    transform: [{translateX: withTiming(rightProgress.value - actualThumbSize.width / 2, {duration: 1})}],
+    zIndex: rightAnimated.value.zIndex,
+  }))
+
+  /**
+   * Update opacity when touching the thumb
+   * This animation should run within 200ms when the thumb is touched or released
+   */
+  const leftOpacityStyle = useAnimatedStyle(() => ({
+    opacity: withTiming(leftAnimated.value.opacity, {
+      duration: DURATION,
+    }),
+  }))
+
+  const rightOpacityStyle = useAnimatedStyle(() => ({
+    opacity: withTiming(rightAnimated.value.opacity, {
+      duration: DURATION,
+    }),
+  }))
+
+  /**
+   * Display the current slider value in real time as the user slides the thumb along the slider track
+   */
+  const leftAnimatedProps = useAnimatedProps(
+    () =>
+      ({
+        text: `${minimumValue + sliderValue.value.left * step}`,
+      } as AnimatedLabelProps),
+  )
+
+  const rightAnimatedProps = useAnimatedProps(
+    () =>
+      ({
+        text: `${minimumValue + sliderValue.value.right * step}`,
+      } as AnimatedLabelProps),
+  )
```

Comment:
same above

Reply:
the line should maximum should be limited to of 80 characters